spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1506,6 +1523,12 @@ When set to a negative value, it defaults to `spring.cloud.stream.instanceCount`
 See "`<<spring-cloud-stream-overview-instance-index-instance-count>>`" for more information.
 +
 Default: `-1`.
+polled.recoverable::
+When set to true (default) exceptions thrown by the polled consumer callback will result in an ErrorMessage sent to the consumer's error channel (and the global Spring Integration errorChannel).
+Set to `false` if, say, you want to requeue messages by throwing `RequeueCurrentMessageException` s.
+See <<polled-errors>>.
```

Comment:
After reading your explanation I now wonder why do we need this property at all? Given that user is in full control of message retrieval and processing should we just send to _error channel_ ***unless*** user throws `RequeueCurrentMessageException`, at which point we will requeue?